File: servidorCanciones/dominio/cancion/dto/CancionAlmacenarDTOInput.go
package dto

type CancionAlmacenarDTOInput struct {
	Titulo          string `json:"titulo"`
	Genero          string `json:"genero"`
	Artista         string `json:"artista"`
	Idioma          string `json:"idioma"`
	AnioLanzamiento int32  `json:"anio_lanzamiento"`
	Duracion        string `json:"duracion"`
}

File: ServidorEnvioCorreos/src/main/java/co/edu/unicauca/infoii/correo/DTOs/CancionAlmacenarDTOInput.java
package co.edu.unicauca.infoii.correo.DTOs;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class CancionAlmacenarDTOInput {

    private String titulo;
    private String artista;
    private String genero;
    private String idioma;
    private String mensaje;
}